app/views/talk/show.html.erb

```diff
@@ -1,2 +1,5 @@
-<div><p class="help-text" style="margin-top: 10px;font-family: 'Junction Light';clear:left;"><%= raw t('talk.show.welcome', :page => @node.latest.title, :url1 => @node.path, :url2 => '/wiki/talk-pages') %></p></div>
+<div><p class="help-text" style="margin-top: 10px;clear:left;">
+  <%= raw t('talk.show.welcome', :page => @node.latest.title, :url1 => @node.path, :url2 => '/wiki/talk-pages') %>
+  | for help, <a href="/chat">visit our chatroom</a>
```

Comment:
Remove newline from here `line 3` and append text at en of line 2 like this `<%= raw t('talk.show.welcome', :page => @node.latest.title, :url1 => @node.path, :url2 => '/wiki/talk-pages') %> | for help, <a href="/chat">visit our chatroom</a>`

and then change line 420 in test to `assert_select 'p', ActionView::Base.full_sanitizer.sanitize(I18n.t('talk.show.welcome', page: nodes(:about).latest.title, url1: nodes(:about).path, url2: '/wiki/talk-pages') + ' | for help, <a href="/chat">visit our chatroom</a>')`

I hope it will resolve the test error.

Reply:
thank you so much @Gauravano